### PR TITLE
Feature/ Current Active task on member profile

### PR DESCRIPTION
**Pull request by sanyogita14**
The current Active task is displayed on the respective member's profile. 

image

### CODE REVIEW

File: components/member-profile/active-task/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from './active-task.module.scss';
+import { timeWas } from '../../../helper-functions/time-was';
+const ActiveTask = (props) => {
```
Comment (sumitd94) [inline components/member-profile/active-task/index.js:5]: NIT: why not destructure `props` here?

```
= ({ taskDetails }) => {}
```

File: components/member-profile/active-task/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        <h3 className={classNames.featureTitle}>{taskDetails.title}</h3>
+        <p className={classNames.prDescription}>{taskDetails.purpose}</p>
+        <div className={classNames.completedData}>
+          <span>Estimated Completion in </span>
+          <p className={classNames.completedDate}>{completedDate}</p>
+        </div>
+      </div>
+    </div>
+  );
+};
```
Comment (sumitd94) [inline components/member-profile/active-task/index.js:21]: let's destructure `taskDetails`
```javascript
const { title, purpose, startedOn, endsOn } = taskDetails;
```
and then use these variables.

File: components/member-profile/contribution-type/index.js [hunk 3]
```diff
@@ -14,8 +15,16 @@ const renderContributions = (contributions, fullName, imageLink, devUser) =>
     />
   ));
 
+const renderActiveTasks = (tasks) => {
+  return tasks.map((task, index) => {
+    if (task.status === 'active') {
```
Comment (sumitd94) [inline components/member-profile/contribution-type/index.js:20]: why are we checking if the status is `active`? I thought the API was returning only Active tasks.

File: pages/[id]/index.js [hunk 4]
```diff
@@ -50,8 +57,11 @@ export async function getServerSideProps(context) {
     const contributionsResponse = await fetch(contributionsURL);
     const contributions = await contributionsResponse.data;
     const imageLink = getImgURL(id, 'img.png');
+    const tasksResponse = await fetch(tasksURL);
+    const tasksData = await tasksResponse.data;
+    const tasks = tasksData.tasks;
```
Comment (sumitd94) [inline pages/[id]/index.js:62]: NIT: can be converted into one line
```javascript
const { tasks } = await tasksResponse.data;
```

**Review by sumitd94 — CHANGES_REQUESTED** [review]
Few changes requested, please.

File: components/member-profile/contribution-type/index.js (hunk 3 above)
Comment (sanyogita14) [reply components/member-profile/contribution-type/index.js:20]: Ok i thought it'd returned all the tasks of that member like even the completed ones... I'll check again

Comment (sumitd94) [reply components/member-profile/contribution-type/index.js:20]: Yes please check once. let me know if the API is not returning only `Active` tasks. Will get in touch with the Backend Engineer and get it changed.

File: components/member-profile/active-task/active-task.module.scss [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+
+@media (max-width: 600px){
+  .container{
+    flex-direction: column;
+  }
+  .taskSection .featureTitle{
+    font-size: 16px;
+  }
+}
```
[the remark this answers is not in the document]
Comment (sanyogita14) [reply components/member-profile/active-task/active-task.module.scss:36]: Yeah okay...I used the same styling as specified in contribution.module.scss. There the font-size is declared in `px`

File: components/member-profile/contribution-type/index.js (hunk 3 above)
Comment (sanyogita14) [reply components/member-profile/contribution-type/index.js:20]: The status of the tasks returned is eother`assigned` or `active` . So should I remove the condition?

File: components/member-profile/active-task/active-task.module.scss (hunk 5 above)
[the remark this answers is not in the document]
Comment (sumitd94) [reply components/member-profile/active-task/active-task.module.scss:36]: yes, we will be refactoring that. but I am making sure whatever new PR's are coming in .. we follow the right practice of using `rem`, `em`.

[the remark this answers is not in the document]
Comment (sanyogita14) [reply components/member-profile/active-task/active-task.module.scss:36]: Okay. I'll make the changes.

File: components/member-profile/active-task/index.js (hunk 1 above)
Comment (sanyogita14) [reply components/member-profile/active-task/index.js:5]: Ok. Didn't think of it. Thanks!

File: components/member-profile/active-task/index.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```
Comment (sumitd94) [inline components/member-profile/active-task/index.js:1]: This import is not needed. By default, we have this imported.

**Review by sumitd94 — CHANGES_REQUESTED** [review]
One small change and we are good to go here.

**Review by sumitd94 — APPROVED** [review]
Thanks for the changes 🥇

**Pull request merged**